.eslintrc

```diff
@@ -24,7 +24,8 @@
     "no-use-before-define": ["error", { "functions": false, "classes": true }],
     "no-var": ["error"],
     "quotes": ["error", "single"],
-    "semi": ["error", "always"]
+    "semi": ["error", "always"],
+    "comma-dangle": ["warn", "only-multiline"]
```

Comment:
No that's correct. But for now I just want to allow it. We'll enforce this rule later when working on #47.